korzh_bot/handlers.py

```diff
@@ -85,6 +85,10 @@ async def post_friends(self, request):
                 "Adding a friend {}. Steam id: {}".format(new_friend.name, steam_id)
             )
             if new_friend not in self.steam_bot.user.friends:
+                if len(self.steam_bot.user.friends) > 150:
```

Comment:
is 150 max limit? if so, why don't we start deleting with like 140? And maybe not delete all friends but like 10. 10 oldest one if possible